pkgs/applications/misc/xppen/generic.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  autoPatchelfHook,
+  qt5,
+  libusb1,
+  pname,
+  version,
+  url,
+  hash,
+}:
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchzip {
+    extension = "tar.gz";
+    inherit url hash;
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    qt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qt5.qtbase
+    libusb1
+  ];
+
+  dontConfigure = true;
+  dontBuild = true;
+  dontCheck = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    rm -r App/usr/lib/pentablet/{lib,platforms,PenTablet.sh}
+    mkdir -p $out/{bin,usr}
+    cp -r App/lib $out/lib
+    cp -r App/usr/share $out/share
+    cp -r App/usr/lib $out/usr/lib
+
+    sed -i 's#/usr/lib/pentablet#/var/lib/pentablet#g' $out/usr/lib/pentablet/PenTablet
```

Comment:
Use substituteInPlace to notice when it fails

Reply:
`PenTablet` is a binary, I don't think it's possible to use here. This is a very hacky way that wouldn't be an issue if we'd create an FSHEnv for it, but IIRC last time that caused more issues than it solved. This is what happens with `substituteInPlace`:
```bash
xppen_4> consumeEntire(): ERROR: Input null bytes, won't process
```
I'll add a comment to make this clear.